nextflow.config

```
@@ -189,7 +190,7 @@ manifest {
     description     = """Demultiplexing pipeline for Illumina sequencing data"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=22.10.1'
-    version         = '1.1.0'
+    version         = '1.1.1'
```

```suggestion
    version         = '1.2.0'
```
This version has additions

> MAJOR version when you make incompatible API changes
    MINOR version when you add functionality in a backwards compatible manner
    PATCH version when you make backwards compatible bug fixes